specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2022-03-01-preview/streamingjobs.json

```diff
@@ -0,0 +1,1126 @@
+      "enum": [
+        "Stop",
+        "Drop"
+      ],
+      "x-ms-enum": {
+        "name": "OutputErrorPolicy",
+        "modelAsString": true
+      }
+    },
+    "CompatibilityLevel": {
+      "type": "string",
+      "description": "Controls certain runtime behaviors of the streaming job.",
+      "enum": [
+        "1.0",
+        "1.2"
+      ],
+      "x-ms-enum": {
+        "name": "CompatibilityLevel",
+        "modelAsString": true
+      }
+    },
+    "JobStorageAccount": {
+      "type": "object",
+      "description": "The properties that are associated with an Azure Storage account with MSI",
+      "allOf": [
+        {
+          "$ref": "./inputs.json#/definitions/StorageAccount"
+        }
+      ],
+      "properties": {
+        "authenticationMode": {
+          "$ref": "../../common/v1/definitions.json#/definitions/AuthenticationMode",
+          "description": "Authentication Mode."
+        }
+      }
+    },
+    "Identity": {
+      "type": "object",
+      "description": "Describes how identity is verified",
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "description": "The tenantId of the identity."
+        },
+        "principalId": {
+          "type": "string",
+          "description": "The principalId of the identity."
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of identity, can be SystemAssigned or UserAssigned."
+        },
+        "userAssignedIdentities": {
+          "type": "object",
+          "description": "The user assigned identities associated with the streaming job resource."
+        }
+      }
+    },
+    "External": {
+      "description": "The storage account where the custom code artifacts are located.",
+      "type": "object",
+      "properties": {
+        "storageAccount": {
+          "$ref": "./inputs.json#/definitions/StorageAccount"
+        },
+        "container": {
+          "type": "string"
+        },
+        "path": {
+          "type": "string"
+        }
+      }
+    },
+    "GetStreamingJobSkuResults": {
+      "type": "object",
+      "description": "Result of the request to get streaming job SKUs.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GetStreamingJobSkuResult"
+          },
+          "description": "The list of available SKUs that the streaming job can use."
+        }
+      }
+    },
+    "GetStreamingJobSkuResult": {
+      "type": "object",
+      "description": "Describes an available SKU information.",
+      "properties": {
+        "resourceType": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of resource the SKU applies to."
```

Comment:
>type of resource [](http://example.com/codeflow?start=30&length=16)

Is this the ARM resource type? If so, could include that in the description, along with whether it is qualified by namespace or not. If the 'type of resource' is something else, can the types be enumerated? If so, consider using a string enum instead of open-ended string.